util/codec/codec.go

```diff
@@ -172,3 +172,145 @@ func DecodeKey(b []byte) ([]interface{}, error) {
 
 	return values, nil
 }
+
+// EncodeValue encodes args to a byte slice which can be decoded by DecodeValue later.
+// It does not guarantee the order for comparison.
+func EncodeValue(args ...interface{}) ([]byte, error) {
```

Comment:
What about varint and uvarint? Should we use it for encodings that does not care about byte order?